go/plumbing/operations/operations_client.go

```diff
@@ -3402,7 +3402,7 @@ func (a *Client) ProvisionSiteTLSCertificate(params *ProvisionSiteTLSCertificate
 }
 
 /*
-PurgeCache purge cache API
+PurgeCache Purges cached content from Netlify's CDN. Support purging by Cache-Tags.
```

Comment:
[pebble] suggestion to tidy up grammar and match response header name verbatim for clarity 

```suggestion
PurgeCache Purges cached content from Netlify's CDN. Supports purging by Cache-Tag.
```